content/docs/start/data-management/index.md

```diff
@@ -15,3 +15,56 @@ We can also build and version pipelines to capture our data workflows stage by
 stage, from raw data and its pre-processing, through feature engineering and ML
 model training, and up to evaluation (performance metrics), visualization, or
 other post-processing.
+
+## Following the Get Started
+
+Although presented in a logical order, each page in this trail is an independent
+chapter you can go to directly. There may be references to other chapters or
+docs, but you can skip them and still get a good idea of the features in
+question, even by only reading page sections.
+
+For better learning, you can also try each step yourself and get to similar
+results. Some of the required steps for this may be inside collapsed sections
+you can click on to expand:
+
+<details>
+
+### Click for an example!
+
+You'll find useful notes and tips throughout our docs such as this one:
+
+<admon type="tip">
+
+The steps and results of some of these chapters are captured in our
+[example-get-started] repo on GitHub. Feel free to clone it and `git checkout`
+any of it's [tags] to explore further.
+
+[example-get-started]: https://github.com/iterative/example-get-started
+[tags]: https://github.com/iterative/example-get-started/tags
```

Comment:
Should we maybe turn the tags that correspond to certain chapters and sections into Releases in the GH repo? I.e. from some https://github.com/iterative/example-get-started/tags to https://github.com/iterative/example-get-started/releases in this case. May be easier to mention releases than all the tags (not so easy to explore and match to the GS).